implementer-guide/setup/migration-tool.md

```diff
@@ -87,21 +93,21 @@ The configuration file that determines the rules for the migration is in JSON-fo
 ### Configuration file rules
 The configuration file describes which assets will be migrated, applying the rules for each asset or collection as shown below: 
 
-  * **User** – the JSON file must contain a single user. If the username exists in the target storage, its data will not be updated, just its ID will be retrieved and will be used when migrating the reports asset. If the user is new to the target storage, it will be inserted as administrator. If the user exists on the target storage and has NO administrator rights, the migration process will abort with explanatory message.
+  * **User** – the JSON file must contain a single user. If the username exists in the target storage, its data will not be updated, just its ID will be retrieved and will be used when migrating the reports asset. If the user is new to the target storage, it will be inserted as administrator. If the user exists on the target storage and has NO administrator rights, the migration process will abort with an explanatory message.
   
   * **Mail Server** – this entry is optional. If provided, it will insert or update the mail server settings on the target storage. 
   
   * **Assets** - defines the assets that will be migrated as follows:
```

Comment:
Now realized that it will be nice if the migration of the Resources and Shared Data Sources assets will also be configurable.
The good thing is that I tested and it's handled by default now, so at least the migration process works correctly.